r/reward_entry/entry.gno

```diff
@@ -0,0 +1,87 @@
+package reward_entry
+
+import (
+	"sort"
+	"std"
+	"time"
+
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/ufmt"
+)
+
+var entries avl.Tree // address -> *RewardEntry
```

Comment:
It prevents a single address from having multiple entries.

We can survive with this condition, but I think it makes sense for such manual entry realm to support multiple entries per address.

Reply:
I don't think it's a good to have multiple entries for single address. 

Btw you can always override points (increase/decrease).

For some reason we want entry for every points earner. we can have something like:
```go
// RewardEntry represents a reward entry
type RewardEntry struct {
	address std.Address
	points  []*Point

	updatedAt time.Time
	updatedBy std.Address
}

type Point struct {
        points  uint64
	reason  string
}
```

Considering the time constrains, I suggest we shouldn't make things unnecessarily complex, unless it is need of the hour